Trees/FenwickTree.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Author: Mohit Kumar
+ * Fedwick Tree Implementation in JavaScript
+ * Fedwick Tree Implementation for finding prefix sum.
+*/
+
+class FenwickTree {
+  constructor (feneickArray, array, n) {
+    for (let i = 1; i <= n; i++) {
+      feneickArray[i] = 0
+    }
+    for (let i = 0; i < n; i++) {
+      this.updateFenwickTree(feneickArray, n, i, array[i])
+    }
+  }
+
+  updateFenwickTree (feneickArray, n, index, value) {
```

Comment:
```suggestion
  update (feneickArray, n, index, value) {
```
`FenwickTree` is in the class name, so no need.

Reply:
Changed `updateFenwickTree` to `update`